### DISCUSSION TITLE
Error missing networkx with QGEP plugin on INTERLIS export, but networkx is  on computer

### DISCUSSION THREAD

**Discussion by sjib** [QGEP/QGEP#689] [Q&A] [answered]
Can networkx be installed in the wrong place when you have different QGIS installations on the computer?

![path_networkx_koni](https://user-images.githubusercontent.com/1265865/131701550-7f2ce773-bb73-4750-8c9a-3a342bfe2116.png)

Do we need to add any additional PATH variables?

**Comment by olivierdalang** [comment]
This is a QGEP issue (qgepqwat2ili doesn't use networkx). Transferring this issue to the QGEP repo.

The easiest is to install it with the OSGeo4W installer, or otherwise `import pip; pip.main(["install", "networkx"])` from the python console in QGIS should also work.

In your case it seems you have it on QGIS 3.16, but maybe not on other versions (when installed with the standalone installer, each QGIS has it's own isolated installation of python).

**Comment by kfischerar** [comment, marked as answer]
for me (windows 10) this works:
pip3 install "networkx"

hope this is the same ;-)

**Reply by sjib** [reply]
OSGeo4W install networkx
![networkx_oswgeo131291197-7b29c115-dd26-491c-9412-7028f61d5ffe](https://user-images.githubusercontent.com/1265865/149361585-7fb2eee7-b7b5-483c-8f59-c23647ec228f.png)

see also https://github.com/QGEP/QGEP/issues/682
